content/deployment/install.md

```diff
@@ -209,304 +147,85 @@ Once you downloaded and verified the release, you are ready to continue with the
-HIGHLY recommended. At minimum you should keep in mind the following.
-
--   CRS does not configure ModSecurity features such as the rule engine,
-    the audit engine, logging etc. This task is part of the ModSecurity
-    initial setup.If you haven't done this yet please check out the
-    [recommended ModSecurity configuration](https://github.com/SpiderLabs/ModSecurity/blob/master/modsecurity.conf-recommended)
--   By default [SecDefaultAction](https://github.com/SpiderLabs/ModSecurity/wiki/Reference-Manual-%28v2.x%29#SecDefaultAction) CRS will redirect to your local domain when an alert is triggered.
-    This may cause redirect loops depending on your configuration. Take
-    some time to decide what you want ModSecurity it do (drop the
-    packet, return a status:403, go to a custom page etc.) when it
-    detects malicious activity.
--   Make sure to configure your anomaly scoring thresholds for more
-    information see [Anomaly]({{< ref "anomaly_scoring.md" >}} "Anomaly")
--   By default ModSecurity looks for lots of issues with different
-    databases and languages, if you are running a specific environment,
-    you probably want to limit this behavior for performance reasons.
--   ModSecurity supports [Project Honeypot](https://www.projecthoneypot.org/) blacklists. This is a
-    great project and all you need to do to leverage it is sign up for
-    an [API key](https://www.projecthoneypot.org/httpbl_api.php) (⚠️ it might be outdated)
--   Do make sure you have added any methods, static resources, content
-    types, or file extensions that your site needs beyond the basic ones
-    listed.
-
-For more information please see the page on
-[configuration]({{< ref "crs.md" >}} "configuration"). Once you have reviewed
-and configured CRS you should rename the file suffix from `.example` to
-`.conf`:
+OWASP CRS contains one setup file that should be reviewed prior to completing setup. The setup file is the only configuration file within the root 'coreruleset-{{< param crs_latest_release >}}' folder and is named `csr-setup.conf.example`. Going through this configuration file and reading what the different options are is HIGHLY recommended. At minimum you should keep in mind the following.
+
+- CRS does not configure features such as the rule engine, the audit engine, logging etc. This task is part of your initial setup. For ModSecurity, if you haven't done this yet please check out the [recommended configuration](https://github.com/SpiderLabs/ModSecurity/blob/master/modsecurity.conf-recommended)
+- By default [SecDefaultAction](https://github.com/SpiderLabs/ModSecurity/wiki/Reference-Manual-%28v2.x%29#SecDefaultAction) CRS will redirect to your local domain when an alert is triggered. This may cause redirect loops depending on your configuration. Take some time to decide what you want ModSecurity it do (drop the packet, return a status:403, go to a custom page etc.) when it detects malicious activity.
```

Comment:
```suggestion
- By default, CRS will redirect to your local domain when an alert is triggered (see [SecDefaultAction](https://github.com/SpiderLabs/ModSecurity/wiki/Reference-Manual-%28v2.x%29#SecDefaultAction)). This may cause redirect loops depending on your configuration. Take some time to decide what you want ModSecurity to do (drop the packet, return a HTTP status code of 403, go to a custom page etc.) when it detects malicious activity.
```